### DISCUSSION TITLE
PS3-Controller / PS4 or another Gamepad

### DISCUSSION THREAD

**Discussion by fabse-hack** [micropython/micropython#10453] [Using MicroPython]
Hello,

Can I use a PS4 controller to control an ESP with Micropython?

I search much and found some libraries for the ps3 / ps4 controller, but it's written in python.
The new intelligent chatbot already lied to me about it :(

Greetings...

**Comment by jimmo** [comment]
My understanding is that the PS4 controllers are Bluetooth Classic, not Bluetooth Low Energy (BLE). MicroPython currently only supports BLE.

**Reply by fabse-hack** [reply]
it's possible to use another gamepad? ... there are some finish repositories ?
